src/io/fastq.rs

```diff
@@ -3,24 +3,103 @@
 // This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! FastQ reading.
+//! Structs and trait to read and write file in FASTA format.
 //!
 //! # Example
 //!
+//! ## Read
+//!
+//! In this example we parse fasta file from stdin and compute some statistics
```

Comment:
In this example, we parse a fastq file from stdin and compute some statistics

Reply:
```suggestion
In this example, we parse a fastq file from stdin and compute some statistics
```